tests/firebird/firebird_integration_test.go

```diff
@@ -0,0 +1,283 @@
+)
+
+func getFirebirdVars(t *testing.T) map[string]any {
+	t.Helper()
+	switch "" {
+	case FirebirdDatabase:
+		t.Fatal("'FIREBIRD_DATABASE' not set")
+	case FirebirdHost:
+		t.Fatal("'FIREBIRD_HOST' not set")
+	case FirebirdPort:
+		t.Fatal("'FIREBIRD_PORT' not set")
+	case FirebirdUser:
+		t.Fatal("'FIREBIRD_USER' not set")
+	case FirebirdPass:
+		t.Fatal("'FIREBIRD_PASS' not set")
+	}
+
+	return map[string]any{
+		"kind":     FirebirdSourceKind,
+		"host":     FirebirdHost,
+		"port":     FirebirdPort,
+		"database": FirebirdDatabase,
+		"user":     FirebirdUser,
+		"password": FirebirdPass,
+	}
+}
+
+func initFirebirdConnection(host, port, user, pass, dbname string) (*sql.DB, error) {
+	dsn := fmt.Sprintf("%s:%s@%s:%s/%s", user, pass, host, port, dbname)
+	db, err := sql.Open("firebirdsql", dsn)
+	if err != nil {
+		return nil, fmt.Errorf("unable to create connection pool: %w", err)
+	}
+	return db, nil
+}
+
+func TestFirebirdToolEndpoints(t *testing.T) {
+	sourceConfig := getFirebirdVars(t)
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	defer cancel()
+
+	var args []string
+
+	db, err := initFirebirdConnection(FirebirdHost, FirebirdPort, FirebirdUser, FirebirdPass, FirebirdDatabase)
+	if err != nil {
+		t.Fatalf("unable to create firebird connection pool: %s", err)
+	}
+	defer db.Close()
+
+	shortUUID := strings.ReplaceAll(uuid.New().String(), "-", "")[:8]
+	tableNameParam := fmt.Sprintf("PARAM_TABLE_%s", shortUUID)
+	tableNameAuth := fmt.Sprintf("AUTH_TABLE_%s", shortUUID)
+	tableNameTemplateParam := fmt.Sprintf("TPARAM_TABLE_%s", shortUUID)
+
+	createParamTableStmts, insertParamTableStmt, paramToolStmt, idParamToolStmt, nameParamToolStmt, arrayToolStmt, paramTestParams := getFirebirdParamToolInfo(tableNameParam)
+	teardownTable1 := setupFirebirdTable(t, ctx, db, createParamTableStmts, insertParamTableStmt, tableNameParam, paramTestParams)
+	defer teardownTable1(t)
+
+	createAuthTableStmts, insertAuthTableStmt, authToolStmt, authTestParams := getFirebirdAuthToolInfo(tableNameAuth)
+	teardownTable2 := setupFirebirdTable(t, ctx, db, createAuthTableStmts, insertAuthTableStmt, tableNameAuth, authTestParams)
+	defer teardownTable2(t)
+
+	createTemplateStmts, insertTemplateStmt, templateTestParams := getFirebirdTemplateParamToolInfo(tableNameTemplateParam)
+	teardownTemplateTable := setupFirebirdTable(t, ctx, db, createTemplateStmts, insertTemplateStmt, tableNameTemplateParam, templateTestParams)
+	defer teardownTemplateTable(t)
+
+	toolsFile := tests.GetFirebirdToolsConfig(sourceConfig, FirebirdToolKind, paramToolStmt, idParamToolStmt, nameParamToolStmt, arrayToolStmt, authToolStmt)
+	toolsFile = tests.AddFirebirdExecuteSqlConfig(t, toolsFile)
+	tmplSelectCombined, tmplSelectFilterCombined := tests.GetFirebirdTmplToolStatement()
+	toolsFile = tests.AddTemplateParamConfig(t, toolsFile, FirebirdToolKind, tmplSelectCombined, tmplSelectFilterCombined, "SELECT * FROM {{.tableName}} ORDER BY ID")
```

Comment:
```suggestion
	toolsFile = tests.AddTemplateParamConfig(t, toolsFile, FirebirdToolKind, tmplSelectCombined, tmplSelectFilterCombined, "")
```

That's the default value, we can just parse an empty string